changedetectionio/tests/test_group.py

```diff
@@ -189,6 +189,7 @@ def test_group_tag_notification(client, live_server):
                                                    "Diff Added: {{diff_added}}\n"
                                                    "Diff Removed: {{diff_removed}}\n"
                                                    "Diff Full: {{diff_full}}\n"
+                                                   "Diff Patch: {{diff_patch}}\n"
```

Comment:
```suggestion
                                                   "Diff as Patch: {{diff_patch}}\n"
```

Reply:
@myhro maybe this one?

Reply:
Sure, I've updated the descriptions. Can you please take a look again?